packages/loot-core/src/client/data-hooks/schedules.tsx

```diff
@@ -23,13 +26,15 @@ function loadStatuses(schedules: ScheduleEntity[], onData) {
 type UseSchedulesArgs = { transform?: (q: Query) => Query };
 type UseSchedulesReturnType = {
   schedules: ScheduleEntity[];
-  statuses: Record<string, ReturnType<typeof getStatus>>;
+  statuses: ScheduleStatuses;
 } | null;
-export function useSchedules({ transform }: UseSchedulesArgs = {}) {
-  let [data, setData] = useState<UseSchedulesReturnType | null>(null);
+export function useSchedules({
+  transform,
+}: UseSchedulesArgs = {}): UseSchedulesReturnType {
+  const [data, setData] = useState<UseSchedulesReturnType>(null);
```

Comment:
<!-- Warning represent user-facing problems. If possible, it's great to follow this kind of comment with a suggestion.  -->
**🔨 warning:** ‏missing `null` 
```suggestion
  const [data, setData] = useState<UseSchedulesReturnType | null>(null);
```

Reply:
Actually, `UseSchedulesReturnType` has null as 'or' type

Reply:
Oh it already has `null`. Thanks for checking!